### ISSUE TITLE
Regression: ``catppuccin.pointerCursor.enable`` linux eval failure

### ISSUE THREAD

**Issue by philipwilk** [catppuccin/nix#381]
It tries to pull in apple-sdk even though the platform is ``x86_64-unknown-linux-gnu`` so eval fails, idk why it's doing this.
[Related bb failure](https://buildbot.fogbox.uk/#/builders/18/builds/5/steps/1/logs/nix_error)

**Comment by h4rldev** [comment]
I have the same issue

**Comment by guhou** [comment]
`catppuccin-cursors` has `pythonPackages.pyside6` as a `nativeBuildInput`; it looks like the regression comes from upstream: https://github.com/NixOS/nixpkgs/issues/356072

**Comment by philipwilk** [comment]
Can confirm if you apply that patch to your sys it works
```nix
{
  meta.description = "python312Packages.pyside6: fix eval on linux #356081";
  url = "https://github.com/NixOS/nixpkgs/pull/356081.patch";
  sha256 = "6PhvfzyRXNlMlgVawBx29yLKDUsoOljWKJs7ryEzCFM=";
}
```
